pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/innvilgelse/OmstillingsstoenadInnvilgelse.kt

```diff
@@ -42,7 +36,10 @@ data class OmstillingsstoenadInnvilgelseDTO(
     val harUtbetaling: Boolean,
     val etterbetaling: OmstillingsstoenadEtterbetaling?,
     val tidligereFamiliepleier: Boolean = false,
-) : FerdigstillingBrevDTO
+    val bosattUtland: Boolean = false,
+) : FerdigstillingBrevDTO {
+    val informasjonOmOmstillingsstoenadData = InformasjonOmOmstillingsstoenadData(tidligereFamiliepleier, bosattUtland)
```

Comment:
Eneste måten jeg ser for å "bundle" to parametere sammen og så sende dem til et vedlegg som brukes på tvers av ulike brev/dto'er

Reply:
Dette er en mye bedre løsning!